components/Editor.js

```diff
@@ -284,7 +282,13 @@ class Editor extends React.Component {
     return (
       <div className="editor">
         <Toolbar>
-          <Themes key={theme} updateTheme={this.updateTheme} theme={theme} />
+          <Themes
+            ref={this.themes}
```

Comment:
What is this `ref` for?

Reply:
Whoops leftover from a previous approach.